shared/tinyusb/tusb_config.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020-2021 Damien P. George
+ * Copyright (c) 2022 Blake W. Felt & Angus Gratton
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+#ifndef MICROPY_INCLUDED_LIB_UTILS_TUSB_CONFIG_H
+#define MICROPY_INCLUDED_LIB_UTILS_TUSB_CONFIG_H
+
+#include <py/mpconfig.h>
+#include "mpconfigport.h"
+
+#if MICROPY_HW_ENABLE_USBDEV
+
+#ifndef MICROPY_HW_USB_STR_MANUF
+#define MICROPY_HW_USB_STR_MANUF "MicroPython"
```

Comment:
This is `MICROPY_HW_USB_MANUFACTURER_STRING` in the stm32 port... we should make them the same.  I do prefer having `STR` before the string name, rather than `_STRING` at the very end.  And then I would prefer the full word "MANUFACTURER" because "MANUF" could be confusing to a newcomer.

Well, whatever we go with, it should be consistent across ports.

Reply:
@dpgeorge Got it. Looks like I saw this name from mimxrt port, where it was added in commit b73073d24.

I'll change to `MICROPY_HW_USB_MANUFACTURER_STRING` for now. If the mimxrt port changes to use this header in the future, we can always add a compatibility #ifdef to pick up the old macro name.